### PR TITLE
Include name argument in service account cmd

### PR DESCRIPTION
**Pull request by smarlowucf**
And use explicit set string option in helm install to handle the account id number as a string.

### CODE REVIEW

**Comment by smarlowucf** [comment]
There are two paths for the account id issue. This one uses `--set-string` to ensure the int is treated as a string. The other option is to set the account id as an int in the helm chart. This is still TBD.

**Review by apozsuse — APPROVED** [review]
LGTM :+1:

**Comment by smarlowucf** [comment]
Sounds like we will make the change in schema https://github.com/neuvector/neuvector-helm/blob/master/charts/core/values.schema.json#L188 so we can drop the `--set-string` change. Marking do not merge for now.

**Comment by smarlowucf** [comment]
@bear454 dropped the set string change since change will happen in upstream chart. This is ready for review and merge.

File: content/neuvector-prime/aws/contents.lr [hunk 1]
```diff
@@ -50,6 +50,7 @@ eksctl create iamserviceaccount \
   --namespace neuvector --cluster $CLUSTER_NAME \
   --role-name $ROLE_NAME --role-only \
   --attach-policy-arn 'arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess' \
+  --name csp \
```
Comment (bear454) [inline content/neuvector-prime/aws/contents.lr:53]: Sorry for the late review, can you explain how this attribute is used?

Comment (smarlowucf) [reply content/neuvector-prime/aws/contents.lr:53]: It's the name of the ServiceAccount in K8s that gets attached to the container role.  https://github.com/neuvector/neuvector-helm/blob/master/charts/core/values.yaml#L51

**Review by bear454 — APPROVED** [review]

**Pull request merged**